swarm/api/http/server.go

```diff
@@ -666,26 +666,6 @@ func (s *Server) updateManifest(key storage.Key, update func(mw *api.ManifestWri
 	if err != nil {
 		return nil, err
 	}
-	s.logDebug("generated manifest %s", key)
+	log.Debug(fmt.Sprintf("generated manifest %s", key))
 	return key, nil
 }
-
-func (s *Server) logDebug(format string, v ...interface{}) {
```

Comment:
I think these methods only introduce indirection without providing with any valuable abstraction.

We are losing the line number where the `log.Error` (or similar log level call) is called, which is very important - this is why I am removing them.